modules/rosa-rhods.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/rosa-available-services.adoc
+
+[id="rosa-rhods_{context}"]
+= Red Hat OpenShift Data Science
+
+Red Hat OpenShift Data Science (RHODS) enables users to integrate data and AI and machine learning software to run end-to-end machine learning workflows. It provides a collection of notebook images with the tools and libraries required to develop and deploy data models. This allows data scientists to easily develop data models, integrate models into applications, and deploy applications using OpenShift. RHODS is available as an add-on to Red Hat managed environments such as Red Hat OpenShift Dedicated and Red Hat OpenShift Service on Amazon Web Services (ROSA).
```

Comment:
* Same comment/question about using "OpenShift" alone here in "using OpenShift".
* Also just checking whether writing out OSD and ROSA in the last sentence is okay, or if there are attributes to use for these?